statistics/update.go

```diff
@@ -190,6 +191,100 @@ func (h *Handle) dumpTableStatDeltaToKV(id int64, delta variable.TableDelta) (bo
 	return updated, errors.Trace(err)
 }
 
+// DumpStatsFeedbackToKV dumps the stats feedback to KV.
```

Comment:
Do we need to add a few metrics for the following operations?

Reply:
Yes, we need to.